src/devices/Mcp23xxx/Mcp23xxx.cs

```diff
@@ -439,11 +438,48 @@ private byte GetMappedAddress(Register register, Port port = Port.PortA, BankSty
             return port == Port.PortA ? address : address += 0x10;
         }
 
+        public void OpenPin(int pinNumber) => SetPinMode(pinNumber, PinMode.Input);
```

Comment:
I'm not sure about this one. If there is no such thing as opening a pin on the Mcp23 devices, then I would probably have this be a a no-op method instead.

Reply:
I was thinking about doing this for now on the bindings that implement this so we can review and update later.  These devices are somewhat an all or nothing.  However, they can be configured to be in a high-impedance state so IMHO this could be considered a closed state.  Maybe keep as no-op for now.